px-data-grid-filterable-mixin.html

```diff
@@ -9,7 +9,16 @@
     static get properties() {
       return {
         /**
-         * If true, turns on advanced filtering.
+         * Set to `true` to enabled advanced filtering. When enabled a filter
+         * button will be shown at the top of the grid. When the user clicks
+         * on the filter button a modal will be shown and the user can
+         * filter visible items using different UI patterns.
```

Comment:
I would say `Set to enable advanced filtering.` or `Whether to enable advanced filtering.` since the whole boolean "true" thing kinda depends on whether you're using binding, HTML, JS, etc. Same comment applies to all boolean descriptions throughout the component...

Commas needed in `When enabled, a` and `When the user clicks on the filter button, a modal`